### PR TITLE
Created 'Role in Tech: Product Manager' outline

### PR DESCRIPTION
**Pull request by raibove**
Fixed Issue #725

### CODE REVIEW

**Comment by Kundan28** [comment]
@raibove Please write more descriptive PR tittle and in the comments write what changes you made in this PR. And mention the issue number in the comments down.

**Comment by Kundan28** [comment]
> @raibove Please write more descriptive PR tittle and in the comments write what changes you made in this PR. And mention the issue number in the comments down.

☝️

**Comment by raibove** [comment]
@Kundan28  I changed the PR and also mentioned the issue in comments
Please check if there are any more changes

File: roles-in-tech/product-manager.md [hunk 1]
```diff
@@ -1,91 +1,110 @@
...
 ### Specific Things To Teach
 
-- Focus on this specific talking point
-- and this
-	- This is a sub-thing about the thing
-- Even more things about the things
-- Even more things about the things
+- What is Product Management?
+- What do Product Managers do?
+- Skills PM should have?
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: PM -> Product Manager

File: roles-in-tech/product-manager.md [hunk 2]
```diff
@@ -1,91 +1,110 @@
...
 
 ### Materials
 
 - [Techtonica Slides - Roles in Tech: Product Management](https://docs.google.com/presentation/d/1BxTU4nugas3Z30kIawpJtLhSBL6b5tF1P-d03Hm_JOI/edit?usp=sharing)
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [ Who are PM? short video (7.53 min)](https://www.youtube.com/watch?v=2dczveSrsv8) - A short video by Ravijot Chugh which explains
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: PM -> Product Manager

File: roles-in-tech/product-manager.md [hunk 3]
```diff
@@ -1,91 +1,110 @@
...
 ### Materials
 
 - [Techtonica Slides - Roles in Tech: Product Management](https://docs.google.com/presentation/d/1BxTU4nugas3Z30kIawpJtLhSBL6b5tF1P-d03Hm_JOI/edit?usp=sharing)
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [ Who are PM? short video (7.53 min)](https://www.youtube.com/watch?v=2dczveSrsv8) - A short video by Ravijot Chugh which explains 
+who are Product Managers and what do they do?
+- [ Product Manager's Role (10 min)](https://www.altexsoft.com/blog/business/product-management-main-stages-and-product-manager-role/) 
+Use this guide to know more about role of Product Manager.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: Which guide ?

File: roles-in-tech/product-manager.md [hunk 4]
```diff
@@ -1,91 +1,110 @@
...
-	- This is a sub-thing about the thing
-- Even more things about the things
-- Even more things about the things
+- What is Product Management?
+- What do Product Managers do?
+- Skills PM should have?
 
 ### Materials
 
 - [Techtonica Slides - Roles in Tech: Product Management](https://docs.google.com/presentation/d/1BxTU4nugas3Z30kIawpJtLhSBL6b5tF1P-d03Hm_JOI/edit?usp=sharing)
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: The slide is not following the formatting guidelines described here: #600

File: roles-in-tech/product-manager.md [hunk 5]
```diff
@@ -1,91 +1,110 @@
...
-Make sure to mention these things:
-
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
-
-
+    Go through the slides given in material section to get an idea about Product Management.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: There's only one slide. So slides -> slide

File: roles-in-tech/product-manager.md [hunk 6]
```diff
@@ -1,91 +1,110 @@
...
+    Product management can be defined as the general business structure within a company that supports and 
+    manages all the activities related to developing, planning, forecasting, marketing and selling a product 
+    all through its lifecycle.
+	
+#### Job of Product Manager
+    Product managers are the official translators within a company, helping commercial teams understand what 
+    the technical, engineering teams mean and vice-versa.
+    Responsibilities that a product manager usually takes are :-
+    - User/market research and analysis
+    - create a solid Product strategy
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: create -> Create

File: roles-in-tech/product-manager.md [hunk 7]
```diff
@@ -1,91 +1,110 @@
...
+    - Learn to Prioritize and re-prioritize various work threads.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: Prioritize -> prioritize

File: roles-in-tech/product-manager.md [hunk 8]
```diff
@@ -1,91 +1,110 @@
...
+    - Create a product and feature roadmap.
+    - Know how to make a product stable and scalable.
+    - Communicate effectively with Designers, Customers, Engineers and clients.  
+    
+    The Role of PM may change depending on type of product and the company stage, etc.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: PM -> Product Manager

File: roles-in-tech/product-manager.md [hunk 9]
```diff
@@ -1,91 +1,110 @@
...
+
+#### Qualities of PM:-
+    - Strategic Thinker
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: Thinker -> thinker

File: roles-in-tech/product-manager.md [hunk 10]
```diff
@@ -1,91 +1,110 @@
...
+    - Proper Planning and Execution
+ 
 ### Common Mistakes / Misconceptions
-
-List things that apprentices might not realize, might assume at first, or should avoid.
-
-- Example
-- Example
+    - Overlook the feedback from customers. 
+    - Use too much of tool on a single product which waste both time and efforts.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: tool -> tools

File: roles-in-tech/product-manager.md [hunk 11]
```diff
@@ -1,91 +1,110 @@
...
+    - Build a Product according to current situation not future needs.
 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+Complete the Links mentioned in the Materials sections. Understand whatever mentioned in those turorials and try to cover it all.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: Complete -> Go through

File: roles-in-tech/product-manager.md [hunk 12]
```diff
@@ -1,91 +1,110 @@
...
+After watching the video and reading the article you will understand  Who are Product Manager and what do they do.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: There are two white space after `understand`.

File: roles-in-tech/product-manager.md [hunk 13]
```diff
@@ -1,91 +1,110 @@
...
 
 ### Supplemental Materials
-- [example website](https://example.com) - Write a very short description of how to use this optional resource.
-- [Other example website](https://otherexample.com) - Write a very short description of how to use this optional resource.
+- [Short Course](https://www.chalkstreet.com/product-management-tutorial-101/) - This is a complete course which gives a brief
+introduction on Product Management.
+- [Slides](https://www.slideshare.net/jysimon/01-pm101-howtocreateproductscustomerslovejysimonjul2014) - This are slides related to
+Product Management.
+- [Qualities of PM](https://www.linkedin.com/pulse/20140410232440-137353-9-qualities-of-a-great-product-manager/) - Skills requiered to be a Product
+Manager are explained on this page.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: No need of the line 110.

**Comment by raibove** [comment]
@Kundan28  I made the changes and also changed the slide according to format

**Comment by Kundan28** [comment]
> @Kundan28 I made the changes and also changed the slide according to format

In Slide no. 2 the regular font size is 20, it should be 18 or 24.

**Comment by Kundan28** [comment]
And in the slide no. 9 the heading font should be Lato.

**Comment by Kundan28** [comment]
The The text in the last line of the slide no. 7 is overlapping with the bar at the bottom.

**Comment by raibove** [comment]
> The The text in the last line of the slide no. 7 is overlapping with the bar at the bottom.

Made the changes. Please review it.

**Comment by Kundan28** [comment]
> Made the changes. Please review it.

There is no Techtonica logo & bar at the bottom in slide no. 5

File: roles-in-tech/product-manager.md [hunk 14]
```diff
@@ -1,91 +1,112 @@
...
+  Product management can be defined as the general business structure within a company that supports and 
+   manages all the activities related to developing, planning, forecasting, marketing and selling a product 
+   all through its lifecycle.
+	
+#### Job of Product Manager
+  
+    Product managers are the official translators within a company, helping commercial teams understand what 
+    the technical, engineering teams mean and vice-versa.
+    
+    Responsibilities that a product manager usually takes are :-
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: Please change to a notation other than code notation.

File: roles-in-tech/product-manager.md [hunk 15]
```diff
@@ -1,91 +1,112 @@
...
+    - Create a solid Product strategy
+    - Learn to prioritize and re-prioritize various work threads.
+    - Create a product and feature roadmap.
+    - Know how to make a product stable and scalable.
+    - Communicate effectively with Designers, Customers, Engineers and clients.  
+    
+   The Role of PM may change depending on type of product and the company stage, etc.
+
+#### Qualities of PM:-
+    - Strategic thinker
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: Same as line 60

File: roles-in-tech/product-manager.md [hunk 16]
```diff
@@ -1,91 +1,112 @@
...
+- What do Product Managers do?
+- Skills PM should have?
 
 ### Materials
 
 - [Techtonica Slides - Roles in Tech: Product Management](https://docs.google.com/presentation/d/1BxTU4nugas3Z30kIawpJtLhSBL6b5tF1P-d03Hm_JOI/edit?usp=sharing)
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [ Who are PM? short video (7.53 min)](https://www.youtube.com/watch?v=2dczveSrsv8) - A short video by Ravijot Chugh which explains
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: Who are PMs

File: roles-in-tech/product-manager.md [hunk 17]
```diff
@@ -1,91 +1,112 @@
...
+ 
 ### Common Mistakes / Misconceptions
-
-List things that apprentices might not realize, might assume at first, or should avoid.
-
-- Example
-- Example
+   
+- Overlook the feedback from customers. 
+- Use too much of tools on a single product which waste both time and efforts.
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: Can you rephrase line 81? I’m not sure what this means

File: roles-in-tech/product-manager.md [hunk 18]
```diff
@@ -1,91 +1,112 @@
...
+- Build a Product according to current situation not future needs.
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: Need comma or “and” after situation

File: roles-in-tech/product-manager.md [hunk 19]
```diff
@@ -1,91 +1,112 @@
...
 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+Go through the Links mentioned in the Materials sections. Understand whatever mentioned in those turorials and try to cover it all.
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: Section,  not sections

File: roles-in-tech/product-manager.md [hunk 20]
```diff
@@ -1,91 +1,112 @@
...
+After watching the video and reading the article you will understand Who are Product Manager and what do they do.
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: Who product managers are

File: roles-in-tech/product-manager.md [hunk 21]
```diff
@@ -1,91 +1,112 @@
...
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+Go through the Links mentioned in the Materials sections. Understand whatever mentioned in those turorials and try to cover it all.
+After watching the video and reading the article you will understand Who are Product Manager and what do they do. 
 
 ### Independent Practice
 
-Class does this thing themselves with specific additional items. This could be alone, with a partner, or small group; but the idea is that it's less guided, more independent.
+Learn to [Identify Market Problem](https://learn.marsdd.com/mars-library/identifying-market-problems/) and to [Prioritize Product Requirement](https://learn.marsdd.com/mars-library/prioritizing-product-requirements/) to Build a Successfull Product.
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: Market problems with an s

File: roles-in-tech/product-manager.md [hunk 22]
```diff
@@ -1,91 +1,112 @@
...
+
 
 ### Challenge
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
+Can you think what kind of challenges does a Product Manageer have to face?
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: You can remove “can you think”

File: roles-in-tech/product-manager.md [hunk 23]
```diff
@@ -1,91 +1,112 @@
...
+ about this topic.
 
-Some ideas: have apprentices summarize to each other, make a cheat sheet, take a quiz, do an assignment, or something else that helps them self-assess their understanding. This exercise should help apprentices determine whether they've met the outline objectives, or if they need to review.
 
 ### Supplemental Materials
-- [example website](https://example.com) - Write a very short description of how to use this optional resource.
-- [Other example website](https://otherexample.com) - Write a very short description of how to use this optional resource.
+- [Short Course](https://www.chalkstreet.com/product-management-tutorial-101/) - This is a complete course which gives a brief introduction on Product Management.
+- [Slides](https://www.slideshare.net/jysimon/01-pm101-howtocreateproductscustomerslovejysimonjul2014) - This are slides related to Product Management.
+- [Qualities of PM](https://www.linkedin.com/pulse/20140410232440-137353-9-qualities-of-a-great-product-manager/) - Skills requiered to be a Product
```
Comment (alodahl) [inline roles-in-tech/product-manager.md]: Required

File: roles-in-tech/product-manager.md [hunk 24]
```diff
@@ -1,91 +1,112 @@
...
-Make sure to mention these things:
-
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
-
-
+   Go through the slide given in material section to get an idea about Product Management.
```
[the remark this answers is not in the document]
Comment (Kundan28) [reply roles-in-tech/product-manager.md]: `Slideshow` could be better. @alodahl @raibove

File: roles-in-tech/product-manager.md (hunk 15 above)
Comment (raibove) [reply roles-in-tech/product-manager.md]: Can you please check the changes I have made. I wasn't clear what you meant by 'Same as line 60'

File: roles-in-tech/product-manager.md (hunk 17 above)
Comment (raibove) [reply roles-in-tech/product-manager.md]: I changed it. I can also remove it if the point is not clear. I also added a new point.

File: roles-in-tech/product-manager.md (hunk 15 above)
Comment (Kundan28) [reply roles-in-tech/product-manager.md]: @alodahl meant to say that the meaning of the line 69 and 60 are same. So you could remove one of them. @raibove

File: roles-in-tech/product-manager.md [hunk 25]
```diff
@@ -77,24 +75,24 @@ Use this guide to know more about role of Product Manager.
...
-Learn to [Identify Market Problem](https://learn.marsdd.com/mars-library/identifying-market-problems/) and to [Prioritize Product Requirement](https://learn.marsdd.com/mars-library/prioritizing-product-requirements/) to Build a Successfull Product.
+Learn to [Identify Market Problems](https://learn.marsdd.com/mars-library/identifying-market-problems/) and to [Prioritize Product Requirement](https://learn.marsdd.com/mars-library/prioritizing-product-requirements/) to Build a Successfull Product.
 
 
 ### Challenge
 
-Can you think what kind of challenges does a Product Manageer have to face? 
+What kind of challenges does a Product Manageer have to face? 
 Check out this [blog](https://productcoalition.com/the-6-biggest-challenges-facing-product-management-teams-in-2018-ef9fd9cfdb9a) 
 to know more.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: Make the content of line 96 and 97 in one line.

File: roles-in-tech/product-manager.md (hunk 15 above)
Comment (alodahl) [reply roles-in-tech/product-manager.md]: No, I mean this has the same problem that I mentioned for line 60.

Comment (alodahl) [reply roles-in-tech/product-manager.md]: It is in code notation but it is not code -please change markdown

Comment (Kundan28) [reply roles-in-tech/product-manager.md]: Oh.

File: roles-in-tech/product-manager.md (hunk 14 above)
Comment (raibove) [reply roles-in-tech/product-manager.md]: I don't know what is code notation. Shall I remove the highlight  I had made to that paragraph. (If so, I made that change.)

**Comment by Kundan28** [comment]
@raibove Please see the comments in the slideshow page no. 12 and fix it.

File: roles-in-tech/product-manager.md (hunk 25 above)
Comment (raibove) [reply roles-in-tech/product-manager.md]: @Kundan28  I made all such sentence in one line

File: roles-in-tech/product-manager.md [hunk 26]
```diff
@@ -16,8 +16,7 @@ Here are links to lessons that should be completed before this lesson:
 
 ### Motivation
 
-It reflects the shift from shipping software to shipping the right software for customers. And it’s the product manager’s job 
-to figure that out. According to Deloitte, customer-centric companies are 60% more profitable than their peers.
+It reflects the shift from shipping software to shipping the right software for customers. And it’s the product manager’s job to figure that out. According to Deloitte, customer-centric companies are 60% more profitable than their peers.
```
Comment (Kundan28) [inline roles-in-tech/product-manager.md]: What was changed in this line ?

File: roles-in-tech/product-manager.md (hunk 15 above)
Comment (alodahl) [reply roles-in-tech/product-manager.md]: @raibove This section and the section starting at line 60 are gray when seen in the markdown preview, and they should not be gray unless it is code.  Please fix.

**Comment by alodahl** [comment]
Great work!  I noticed your commit messages are mainly "* Update product-manager.md".
For the future, when you write commit messages, be sure to briefly decribe what you actually did, like "fixed typo in line 3" or "replaced dead link" so that if you need to go back to a certain version for some reason, you'll have some clues!

**Comment by raibove** [comment]
> Great work! I noticed your commit messages are mainly "* Update product-manager.md".
> For the future, when you write commit messages, be sure to briefly decribe what you actually did, like "fixed typo in line 3" or "replaced dead link" so that if you need to go back to a certain version for some reason, you'll have some clues!

Will do it next time. This was first time I made a contribution.
